tests/phpunit/Configuration/ConfigurationFactoryTest.php

```diff
@@ -793,7 +844,89 @@ private static function createValueForInitialTestsFrameworkOptions(
             null,
             false,
             null,
+        ];
+    }
+
+    /**
+     * @param array<string,Mutator> $expectedMutators
+     */
+    private static function createValueForMutators(
+        array $configMutators,
+        ?string $inputMutators,
+        array $expectedMutators
+    ): array {
+        return [
+            new SchemaConfiguration(
+                '/path/to/infection.json',
+                null,
+                new Source([], []),
+                new Logs(
+                    null,
+                    null,
+                    null,
+                    null,
+                    null
+                ),
+                null,
+                new PhpUnit(null, null),
+                $configMutators,
+                null,
+                null,
+                null,
+                null
+            ),
+            null,
+            null,
+            'none',
+            false,
+            false,
+            'progress',
+            false,
+            false,
+            null,
+            false,
+            null,
+            $inputMutators,
+            null,
+            null,
+            10,
+            new Source([], []),
+            new Logs(
+                null,
+                null,
+                null,
+                null,
+                null
+            ),
+            'none',
+            sys_get_temp_dir(),
+            new PhpUnit('/path/to', null),
+            $expectedMutators,
+            null,
+            null,
+            null,
+            null,
+            null,
+            false,
+            false,
+            'progress',
+            false,
+            false,
+            null,
+            false,
             null,
         ];
     }
+
+    /**
+     * @return array<string, Mutator>
+     */
+    private static function getDefaultMutators(): array
```

Comment:
not found of this but could find a better way for now. Indeed here we just want the default mutators listed and since it is a bit expansive to produce, I'm storing it in a static class. However right now providers are evaluated before anything else so this cannot be moved to `setUpBeforeClass()`. I can however unset it afterwards